game/src/ui/Menu/Settings.tscn

```diff
@@ -1,12 +1,16 @@
-[gd_scene load_steps=9 format=2]
+[gd_scene load_steps=10 format=2]
 
 [ext_resource path="res://assets/font/arcadeclassic/ARCADECLASSIC.TTF" type="DynamicFontData" id=1]
 [ext_resource path="res://src/Audio/BgMusicPlayer.tscn" type="PackedScene" id=2]
 [ext_resource path="res://assets/img/Screens/04(1).jpg" type="Texture" id=3]
+[ext_resource path="res://src/ui/gameplay/ResetExit/Reset.tscn" type="PackedScene" id=4]
 
 [sub_resource type="GDScript" id=1]
 script/source = "extends Control
 
+var from_menu = true
```

Comment:
personally i would do only one boolean since from_menu === !from_game, but i'm okay leaving this here

Reply:
lmao yes im dumb

Reply:
fixed.